packages/apollo-server-core/src/plugin/usageReporting/__tests__/plugin.test.ts

```diff
@@ -23,11 +31,14 @@ describe('end-to-end', () => {
     expectReport = true,
     query,
     operationName,
+    schemaShouldBeInstrumented = true,
   }: {
     pluginOptions?: ApolloServerPluginUsageReportingOptions<any>;
     expectReport?: boolean;
     query?: string;
     operationName?: string | null;
+    // null means no expectation because it's not deterministic
+    schemaShouldBeInstrumented?: boolean | null;
```

Comment:
Can this ever be `undefined`? Looks like it's defaulted above
```suggestion
    schemaShouldBeInstrumented: boolean | null;
```

Reply:
The `?` here is more like "key can be left out" than "can also be undefined" (though I think in TS these days it kinda means both and they are trying to migrate away from that). Applying your change means that the calls that don't specify schemaShouldBeInstrumented fail to build.  TS properly determines that schemaShouldBeInstrumented is `boolean | null` (not undefined) inside the function so I think I'm doing it correctly.